go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/fatih/color v1.13.0
 	github.com/google/uuid v1.3.0
 	github.com/mhelmich/go-archiver v0.0.0-20220110024759-0bc40b00af39
-	github.com/nucleuscloud/mgmt-api v0.0.42
+	github.com/nucleuscloud/mgmt-api v0.0.63-alisha1
```

Comment:
I think this goes without being said, but let's make sure this gets updated with a real base tag before this is merged.

Reply:
I will change this before I merge. I need to get my mgmt api merged first